base/path.jl

```diff
@@ -165,10 +165,17 @@ See also: [`basename`](@ref)
 
 Get the file name part of a path.
 
+Note that the result of this function is different from the Unix `basename` program, where trailing
+slashes are ignored, i.e. `basename` for `/foo/bar/` returns `bar`, whereas  the `basename` function returns
+an empty string `""`.
```

Comment:
```suggestion
!!! note
    This function differs slightly from the Unix `basename` program, where trailing
    slashes are ignored, i.e. `$ basename /foo/bar/` returns `bar`, whereas  Julia returns an empty string `""`.
```

Reply:
I guess that didn't work because it thinks it's interpolation syntax? 
https://build.julialang.org/#/builders/39/builds/3391/steps/3/logs/stdio